app/main.py

```diff
@@ -14,10 +14,7 @@ def __repr__(self):
 
 class Herbivore(Animal):
     def hide(self):
-        if not self.hidden:
-            self.hidden = True
-        else:
-            self.hidden = False
+        self.hidden = False if self.hidden else True
```

Comment:
ok it's batter but you can just do it `self.hidden = not self.hiden` it's same, but without unnecessaty condition.